eng/build.ps1

```diff
@@ -14,7 +14,7 @@ Param(
   [string]$arch = [System.Runtime.InteropServices.RuntimeInformation]::ProcessArchitecture.ToString().ToLowerInvariant(),
   [string]$subsetCategory,
   [string]$subset,
-  [ValidateSet("Debug","Release","Checked")][string]$runtimeConfiguration,
+  [ValidateSet("Debug","Release","Checked")][string]$runtimeConfiguration = "Debug",
```

Comment:
Why set a default? Instead you could check if it is set before setting the environment variable below.

This will need to be one more place that will need to change when #33240 is fixed.

Reply:
Mainly for consistency. $configuration has a default value as well.